From d4eb89998d0d659b2f8a8279cf3db96b312d546f Mon Sep 17 00:00:00 2001
From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Fri, 15 Apr 2016 11:49:55 +0200
Subject: [PATCH] media: switch source bin fakesink to PAUSED synchronously

Connecting the source bin to the rest if the pipeline while it's still
in the process of transition to PLAYING can block the pipeline.

Backport of Pidgin 3 commit e382df8b8ab4.
---
 libpurple/mediamanager.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libpurple/mediamanager.c b/libpurple/mediamanager.c
index a5702b5..036a2dc 100644
--- a/libpurple/mediamanager.c
+++ b/libpurple/mediamanager.c
@@ -1172,6 +1172,7 @@ purple_media_manager_get_element(PurpleMediaManager *manager,
 			 */
 			fakesink = gst_element_factory_make("fakesink", NULL);
 			g_object_set(fakesink,
+				"async", FALSE,
 				"sync", FALSE,
 				"enable-last-sample", FALSE,
 				NULL);
-- 
2.7.4

